wire-ios/Wire-iOS/Sources/Helpers/TmpFiles/TMPFilesManager.swift

```diff
@@ -0,0 +1,35 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+protocol PTMPFilesManager {
+    func removeTemporaryData()
+}
```

Comment:
Does the P stand for Protocol? I suggest to follow the convention: `SomeClassInterface` and the implementer `SomeClass: SomeClassInterface`.

For naming I suggest to avoid acronyms and prefer "service" over "manager":

```swift
protocol TemporaryFileServiceInterface {
    
    func deleteTemporaryData()

}

final class TemporaryFileService: TemporaryFileServiceManager {

    ...

}

Reply:
Yes P stands for protocol here. I took it from C# or Java I believe where the naming is for example `ISomeService` in which I stands for interface.